components/globalSettings.tsx

```diff
@@ -64,6 +86,24 @@ export const setUrl = (url: string) => {
     });
 };
 
+export const getSecureConnection = (callback: (e: boolean | null) => void) => {
```

Comment:
I would move this. Multiple exports in component modules is a bit confusing imho. These are utility functions.